docs/conf.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'airflow-declarative'
```

Comment:
This metadata looks similar to project ones. Can we reuse that to avoid duplicity of project information?

Reply:
It would create a tight coupling between the package metadata and the docs (i.e. it might become troublesome to compile docs without having the package metadata around). The building pipeline at RTD concerns me most. Do you believe that this is worth pursuing?

Reply:
Yes, because docs are tight coupled with the project itself. RTD should handle that without issues since otherwise how autodoc suppose to be work at all?

Reply:
> otherwise how autodoc suppose to be work at all

I suppose autodoc simply reads the sources of the specified modules as files on the filesystem, which doesn't require the package metadata.

I'll see what I can do, but I feel that it would be easier to just keep these 3 lines rather than introduce the code reading the project's metadata (which will definitely be longer than 3 lines).

Reply:
Pushed. I'm not sure that this is what you had in mind, but at least it takes the metadata from the project.

I decided to use `subprocess.run` for getting values from `setup.py` instead of, say, `subprocess.check_output`, because it looks cleaner (the latter is missing the `cwd` kwarg, which, I suppose, would be required once the metadata would be moved to setup.cfg). The consequence is that it made the docs compatible with Python >=3.5 only.